### PR TITLE
Add an explainer

### PR DESCRIPTION
**Pull request by mprorock**
obviously we will build on this and make it better

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+Verifiable Credentials (VCs) permit several forms of providing security around
+the data model. [JWTs](https://www.rfc-editor.org/rfc/rfc7519) provide a
+widely deployed approach to signing information. This specification
+provides the details necessary to utilize JWTs as a securing mechanism
+for verifiable credentials.
+
+## Authors
+
+* Michael Jones
```
Comment (mprorock) [inline explainer.md:21]:
```suggestion
* Michael B. Jones
```

File: explainer.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+model that enables the issuance, sharing, and verification of digital
+credentials in a secure and interoperable manner. These credentials
+provide a way for individuals, organizations, or entities to digitally
+represent and share their qualifications, attributes, or other relevant
+information. Verifiable Credentials are designed to enhance trust,
+privacy, and control in digital interactions by allowing the owner of
+the credentials to control how their information is shared and verified.
+
+Verifiable Credentials (VCs) permit several forms of providing security around
+the data model. [JWTs](https://www.rfc-editor.org/rfc/rfc7519) provide a
```
Comment (mprorock) [inline explainer.md:14]:
```suggestion
Multiple different methods can be used to cryptographically secure 
content encoded using the Verifiable Credentials (VCs) data model. 
[JWTs](https://www.rfc-editor.org/rfc/rfc7519) provide a
```

**Comment by mprorock** [comment]
@selfissued we are charted explicity for core data model - I am fine to expand the explainer later and have noted the valid concern raised here in #99 - can we get this approved and merged as is to be able to start horizontal review with tag per issue #97 ? 

we cannot do this until explainer is merged.

**Review by OR13 — APPROVED** [review]

File: explainer.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (OR13) [reply explainer.md]: > Before we check this in, I believe we need to describe how VC-JWT payloads can use native JWT claims and need not use JSON-LD.

You are suggesting this PR needs to be merged or closed :https://github.com/w3c/vc-jwt/pull/88

Before we can add this explain and start our horizontal review?

I agree.

**Review by TallTed — CHANGES_REQUESTED** [review]
Minor tweaks and corrections for clarity

**Review by selfissued — APPROVED** [review]
At some point, I believe we need to describe how VC-JWT payloads can use native JWT claims and need not use JSON-LD.

**Comment by OR13** [comment]
Multiple approvals, changes requested and implemented, merging.

**Comment by TallTed** [comment]
> [@selfissued] At some point, I believe we need to describe how VC-JWT payloads can use native JWT claims and need not use JSON-LD.

I suggest you create an Issue and/or PR for this.

**Comment by andresuribe87** [comment]
> > [@selfissued] At some point, I believe we need to describe how VC-JWT payloads can use native JWT claims and need not use JSON-LD.
> 
> I suggest you create an Issue and/or PR for this.

Perhaps we can use #98 to track

**Pull request merged**